src/sql/workbench/contrib/welcome/page/browser/gettingStartedTour.css

```diff
@@ -0,0 +1,371 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads_tour:focus {
+	outline: 0 none !important;
+}
+
+.monaco-workbench>.ads_tour {
+	position: absolute;
+	top: 0;
+	left: 0;
+	height: 100%;
+	width: 100%;
+	z-index: 9999;
+	font-size: 1em;
+	transition: font-size .25s;
+	outline: 0;
+}
+
+#workbench\.parts\.editor {
+	transition: filter .25s, opacity .2s;
+}
+
+.monaco-workbench.blur-background #workbench\.parts\.editor, .monaco-workbench.blur-background #workbench\.parts\.panel, .monaco-workbench.blur-background #workbench\.parts\.sidebar {
+	filter: blur(2px);
+	opacity: .4;
+}
+
+.hc-black .monaco-workbench.blur-background #workbench\.parts\.editor, .hc-black .monaco-workbench.blur-background #workbench\.parts\.panel, .hc-black .monaco-workbench.blur-background #workbench\.parts\.sidebar {
```

Comment:
It'd be great if you could format this so that separate selectors are each on their own line - makes it a lot easier to read

Reply:
I believe this has been fixed